bagitobjecttransfer/recordtransfer/views.py

```diff
@@ -382,9 +453,8 @@ def get_form_initial(self, step):
         """
         initial = self.initial_dict.get(step, {})
 
-        resume_id = self.request.GET.get("resume_transfer", None)
-        if resume_id is not None:
-            transfer = SavedTransfer.objects.filter(user=self.request.user, id=resume_id).first()
+        if self.in_progress_uuid:
+            transfer = get_in_progress_submission(self.request.user, self.in_progress_uuid)
```

Comment:
```suggestion
            transfer = InProgressSubmission.objects.filter(user=self.request.user, uuid=self.in_progress_uuid).first()
```

Reply:
Resolved by:
[Refactored view to use InProgressSubmssion instance variable rather t…](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/240/commits/ce4b0fa60c2c9ffb80c8229683ae4799309f983b)